src/node_debug_options.cc

```diff
@@ -94,18 +94,18 @@ bool DebugOptions::ParseOption(const std::string& option) {
     argument = option.substr(pos + 1);
   }
 
+  // Note that --debug-port and --debug-brk are undocumented, but to be
+  // supported until 7.x is no longer supported, not even in LTS (see #12364).
```

Comment:
It sounds like some people think `--debug-brk` should exist forever, or even replace `--inspect-brk`, I'll fix up the comment once consensus is reached.